Buổi 10/homework.js

```diff
@@ -0,0 +1,146 @@
+//Bài 1
+const Multi = number =>{

```

Comment:
1. Chú ý dấu cách
2. Tên hàm phải là động từ, theo quy tắc camel
Và đọc bộ luật sau đây: https://viblo.asia/p/javascript-style-guide-and-coding-conventions-924lJqVmZPM

```suggestion
const Multi = number => {

```